.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "proto/src/main/proto"]
+	path = proto/src/main/proto
+	url = https://github.com/pavolloffay/opentelemetry-proto.git
```

Comment:
@bogdandrutu I have marked this PR as WIP with depends on https://github.com/open-telemetry/opentelemetry-proto/pull/26.

Since this is merged I will create an issue to change to upsream repo once it's ready